includes/sanitizers/class-amp-form-sanitizer.php

```diff
@@ -19,6 +20,29 @@
  */
 class AMP_Form_Sanitizer extends AMP_Base_Sanitizer {
 
+	/**
+	 * Validation error code when opting-in to native POST forms and one is encountered.
+	 */
+	const FORM_HAS_POST_METHOD = 'FORM_HAS_POST_METHOD';
```

Comment:
Er, actually perhaps `FORM_WITH_POST_METHOD_HAS_ACTION_ATTRIBUTE` would be more more accurate.

Reply:
Fixed in cf8c16b